internal/repositories/github/service.go

```diff
@@ -134,3 +213,75 @@ func (r *repositoryService) deleteRepository(ctx context.Context, client ghclien
 
 	return nil
 }
+
+func (r *repositoryService) pushReconcilerEvent(pbRepo *pb.Repository, projectID uuid.UUID) error {
+	log.Printf("publishing register event for repository: %s/%s", pbRepo.Owner, pbRepo.Name)
```

Comment:
Again, `zerolog.Ctx(ctx)` will give us additional information, and we should prefer it in new code.